site/content/en/docs/Examples/_index.md

```diff
@@ -46,7 +46,11 @@ These are all examples of simple game server implementations, that integrate the
 - {{< ghlink href="examples/allocation-endpoint" >}}Allocation Endpoint Example{{< /ghlink >}} - An Agones clusters 
   aware proxy, built on Google Cloud.
 
-## Integration with Open Match
+## Integrations with other projects
 
 * [googleforgames/space-agon](https://github.com/googleforgames/space-agon) - Space Agon is a demo of Agones and 
   Open Match with a browser based game.
+
+## Global Scale Demo
+
```

Comment:
```suggestion
```

Reply:
Let's  get rid of this header, we can put the entry for Global Scale Demo right below Space Agon.

Reply:
Global multiplayer demo link is added below the Space Agon